packages/frontend/src/Layout.tsx

```diff
@@ -0,0 +1,21 @@
+import { Outlet } from 'react-router-dom';
+import Nav from './components/Nav';
+
+/**
+ * This is the layout component that will be used throughout the website
+ *
+ * @argument noPadding    specifies wether to include padding for the page,
+ * defaults to including padding
+ */
+export const Layout = ({ noPadding = false }) => {
+  const containerClasses = noPadding ? '' : 'px-10 py-12';
+
+  return (
+    <main className="bg-grey-1 min-h-screen">
+      <Nav />
+      <div className={containerClasses}>
+        <Outlet />
+      </div>
+    </main>
+  );
+};
```

Comment:
Here, `<Outlet />` will be replaced by whatever inside the page.  If you go to writing task 1 page, the content of the page will be placed in place of `<Outlet />`, and same goes for other pages.

This is very helpful, as now, not every page has to include `Nav` and include padding to the screen.